### PR TITLE
Remove Salt configuration files and document Dokku / Tequila options.

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

**Comment by mlavin** [comment]
Should we go ahead and remove the fabfile as well? I think it pretty much assumes the project is using Salt for all of the commands.

File: docs/dev/provisioning.rst [hunk 1]
```diff
@@ -24,374 +24,47 @@ These services can configured to run together on a single machine or on differen
...
-    from . import load_env
-    load_env.load_env()
-
-You should now be able to run the worker locally via (once you've added ``celery`` to your
-``requirements/base.txt`` and installed it)::
+Provisioning Options
+--------------------
 
-    celery -A {{ project_name }} worker
+`Titles are targets, too`_.
```
Comment (vkurup) [inline docs/dev/provisioning.rst]: Is this meant to be here? ^

File: docs/dev/updates.rst [hunk 2]
```diff
@@ -15,143 +15,4 @@ Salt and the related states used to provision our servers.
 What Changed
 ------------
 
-Postgres
```
Comment (vkurup) [inline docs/dev/updates.rst]: My preference would be to remove this whole file, as it refers to 'updates' in the project template that are now obsolete.

File: docs/dev/provisioning.rst (hunk 1 above)
Comment (ironfroggy) [reply docs/dev/provisioning.rst]: nope!

File: docs/dev/updates.rst (hunk 2 above)
Comment (ironfroggy) [reply docs/dev/updates.rst]: deletin'

**Pull request merged**